bundle/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Flamingos2Epics.scala

```diff
@@ -99,26 +100,34 @@ object Flamingos2Epics {
 
 }
 
-object Flamingos2EpicsInitializer {
+object Flamingos2Epics extends EpicsSystem {
+  
   val Log = Logger.getLogger(getClass.getName)
   val CA_CONFIG_FILE = "/Flamingos2.xml"
+  
+  // Still using a var and null, but at least now it's hidden. Attempts to access the single instance will
+  // now result in an Exception with a meaningful message, instead of a NullPointerExecption
+  private var instanceInternal: Flamingos2Epics = null
+  lazy val instance: Flamingos2Epics = {
+    if (instanceInternal == null) {
+      throw new Exception("Attempt to reference Flamingos2Epics single instance before initialization.")
+    }
+    instanceInternal
+  }
```

Comment:
I haven't been following this closely but this seems a bit unusual at least.  Is there a reason to not just make a `Flamingos2Epics` when you need it?

If you are going to do this though, you could hide the `null` with (assuming scalaz syntax is imported):

````scala
private var instanceInternal = Option.empty[Flamingos2Epics]

def instance: Flamingos2Epics =
  instanceInternal | throw new Exception("...")
````